### PR TITLE
Set-up NotificationResponder Notes

### PR DESCRIPTION
**Pull request by johnno1962**
Hi @mattmassicotte, All the while I'd been missing out on what the LSP server was trying to tell me. I figure the README.md could do with a quick note about how to set-up a NotificationResponder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,10 +18,13 @@ let host = LanguageServerProcessHost(path: executablePath, arguments: [],
     environemnt: [/* the environment your lsp server requires e.g. PATH */])
 
 host.start { (server) in
-    guard let server = server else {
+    guard let server = server as? JSONRPCLanguageServer else {
```
Comment (mattmassicotte) [inline README.md]: Why the cast here?

Comment (johnno1962) [reply README.md]: You're right. its not needed, I've force pushed an update.

**Comment by mattmassicotte** [comment]
Looks good!

**Comment by johnno1962** [comment]
Cheers!

**Pull request merged**